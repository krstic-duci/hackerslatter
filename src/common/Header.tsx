import styles from "styles/common/header.module.scss";

const Header = () => {
  return (
    <header className={`${styles["header-wrapper"]} text-center`}>
      <h1 className="text-monospace text-white font-weight-bold">
        Hackerslatter
      </h1>
    </header>
  );
};

export default Header;
