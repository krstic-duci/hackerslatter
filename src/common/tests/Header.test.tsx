import renderer from "react-test-renderer";

import Header from "common/Header";

describe("Header snapshot test", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Header />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
