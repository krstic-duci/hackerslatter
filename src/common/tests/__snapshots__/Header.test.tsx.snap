// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot test should match snapshot 1`] = `
<header
  className="header-wrapper text-center"
>
  <h1
    className="text-monospace text-white font-weight-bold"
  >
    Hackerslatter
  </h1>
</header>
`;
